bindings/Python/CMakeLists.txt

```diff
@@ -13,16 +13,29 @@ if(ADIOS2_HAVE_MPI)
   target_link_libraries(adios2py PRIVATE PythonModule::mpi4py)
 endif()
 
-set(CMAKE_INSTALL_PYTHONDIR
-  "${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages"
+if(WIN32)
+  set(CMAKE_INSTALL_PYTHONDIR_DEFAULT
+    "${CMAKE_INSTALL_LIBDIR}\\site-packages"
+  )
+else()
+  set(CMAKE_INSTALL_PYTHONDIR_DEFAULT
+    "${CMAKE_INSTALL_LIBDIR}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages"
+  )
+endif()
+set(CMAKE_INSTALL_PYTHONDIR "${CMAKE_INSTALL_PYTHONDIR_DEFAULT}"
```

Comment:
Fix: Now overwriteable from the command line.

Reply:
Rather than hard coding this with a guess you could use introspection:
```cmake
execute_process(
  COMMAND
    ${PYTHON_EXECUTABLE}
      -c [[
import sys
from distutils import sysconfig
sitedir = sysconfig.get_python_lib(plat_specific=True)
if sitedir.startswith(sys.prefix):
    sitedir = sitedir[len(sys.prefix)+1:]

print(sitedir)
]]
  OUTPUT_VARIABLE CMAKE_INSTALL_PYTHONDIR_DEFAULT
)
```

Also be sure to mark the cache variable as advanced.